yeelight/flow.py

```diff
@@ -67,7 +71,7 @@ def as_list(self):
         :rtype: list
         """
         # Duration must be at least 50, otherwise there's an error.
-        return [max(50, self.duration), self._mode, self._value, self.brightness]
+        return [max(50, self.duration), self._mode, self._value, int(self.brightness)]
```

Comment:
No, I didn't mean the end app, I meant each of the classes that inherit this. But, screw it, I guess this is the best place, since this is the only place we access this attribute anyway.